tests/nlu/base/test_evaluation.py

```diff
@@ -716,12 +716,16 @@ def test_get_evaluation_metrics(
 def test_nlu_comparison(tmpdir):
     configs = [
         NLU_DEFAULT_CONFIG_PATH,
-        "sample_configs/config_supervised_embeddings.yml",
+        "sample_configs/config_convert_response_selector.yml",
     ]
     output = tmpdir.strpath
 
     compare_nlu_models(
-        configs, DEFAULT_DATA_PATH, output, runs=2, exclusion_percentages=[50, 80]
+        configs,
+        "data/test_evaluations/nlu_nlg",
```

Comment:
Can you assign this path to a constant variable. Easier to maintain.

Reply:
sure